netlify.toml

```diff
@@ -46,3 +48,16 @@
   from = "/*"
   to = "/404/"
   status = 404
+[build]
+  base = "content"
+  publish = "public"
+  command = """\
+    export DART_SASS_TARBALL="dart-sass-${DART_SASS_VERSION}-linux-x64.tar.gz" && \
+    (cd /tmp && \
+     curl -LJO ${DART_SASS_URL}/${DART_SASS_VERSION}/${DART_SASS_TARBALL} && \
+     tar -xf ${DART_SASS_TARBALL} && \
+     rm ${DART_SASS_TARBALL}) && \
+    export PATH=/tmp/dart-sass:$PATH && \
+    (cd ../.. && ln -s repo scientific-python-hugo-theme) && \
+    (cd .. && make ${BUILD_TARGET})
```

Comment:
These should be the relevant lines. We do not use the Makefile (having switched it to `noxfile.py`). Could you install `nox` with `pip` here and run `nox -s html`?